packages/bento-design-system/src/Table/createTable.tsx

```diff
@@ -71,9 +71,22 @@ type SortFn<C extends ReadonlyArray<ColumnType<string, {}, any>>> = (
 
 type SortingProps<C extends ReadonlyArray<ColumnType<string, {}, any>>> =
   | {
-      /** This function must be memoized to avoid infinite re-renderings */
+      /**
+       * `customSorting` is meant to be used to provide a custom sorting logic
+       * to the table. It supports cross-columns comparison.
+       *
+       * The sorting is still performed by the table.
+       *
+       * This function must be memoized to avoid infinite re-renderings */
```

Comment:
```suggestion
      /**
       * `customSorting` can be used to customize the sorting logic of the table. It supports cross-columns comparison.
       *
       * The sorting is still performed by the table, in an uncontrolled fashion.
       *
       * This function must be memoized to avoid infinite re-renderings */
```